test/sinatra/router_test.rb

```diff
@@ -1,4 +1,3 @@
-require "minitest/spec"
```

Comment:
Caused a circular require warning, and all styles of testing still work without it, so removed.